omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,61 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',function ($scope,TagService,ngDialog) {
+    $scope.init = function(personUuid) {
+            $scope.thisPersonUuid = personUuid;
```

Comment:
personUuid is what we obtain from the page and what we are sending to the angular app.
https://github.com/openmrs/openmrs-module-tag/pull/7/files#diff-f42d820b44a0dfe6d4a7205d69cb0272R13